src/components/molecules/CLICommands/CLICommands.tsx

```diff
@@ -36,46 +37,53 @@ const testSuiteScripts: CLIScript[] = [
   {
     label: 'Run test suite',
     command: (name: string) => `kubectl testkube run testsuite ${name}`,
+    filename: 'run-test-suite.sh',
   },
   {
     label: 'Delete test suite',
     command: (name: string) => `kubectl testkube delete testsuite ${name}`,
+    filename: 'delete-test-suite.sh',
   },
 ];
 
 const testScripts: CLIScript[] = [
   {
     label: 'Run test',
     command: (name: string) => `kubectl testkube run test ${name}`,
+    filename: 'run-test.sh',
   },
   {
     label: 'Get test',
     command: (name: string) => `kubectl testkube get test ${name}`,
+    filename: 'get-test.sh',
   },
   {
     label: 'List executions',
     command: (name: string) => `kubectl testkube get executions --test ${name}`,
+    filename: 'list-executions.sh',
   },
   {
     label: 'Delete test',
     command: (name: string) => `kubectl testkube delete test ${name}`,
+    filename: 'delete-executions.sh',
   },
 ];
 
 const executionsScripts: CLIScript[] = [
   {
     label: 'Get execution',
     command: (name: string) => `kubectl testkube get execution ${name}`,
+    filename: 'get-executions.sh',
   },
   {
     label: 'Watch execution',
     command: (name: string) => `kubectl testkube watch execution ${name}`,
-    modify: 'isFinished',
+    filename: 'watch-executions.sh',
   },
   {
     label: 'Download artifacts',
     command: (name: string) => `kubectl testkube download artifacts ${name}`,
-    modify: 'canHaveArtifacts',
```

Comment:
What is a point of removing this modify prop, isn't it used just below?
I can't tell exactly about this block, but as first look, it shouldn't be removed

Reply:
You're right this in accidental change that shouldn't have happened. Reverting back.